README.rst

```diff
@@ -15,6 +15,14 @@ command and for specific subcommands through :code:`dcos <subcommand> --help`.
 Additional documentation for the CLI and for the DCOS in general is available
 in the `Mesosphere docs`_.
 
+To run the DCOS CLI as a container, you can pull it from the docker hub with::
+
+    docker run -i -t mesosphere/dcos-cli <url of your cluster>
```

Comment:
I thought that we agreed that we wanted to run this as: `docker run -i -t -v ... mesosphere/dcos-cli dcos ...`.